### PR TITLE
PLANET-7082 Convert Campaign pattern layout into a Block Template

### PR DESCRIPTION
**Pull request by mleray**
### Description

See [PLANET-7082](https://jira.greenpeace.org/browse/PLANET-7082)

### Testing

You should still be able to add the Campaign pattern layout from the `Layouts` category, and it should look just like before.

### CODE REVIEW

File: assets/src/block-templates/page-header/block.json [hunk 1]
```diff
@@ -6,8 +6,8 @@
   "category": "planet4-block-templates",
   "textdomain": "planet4-blocks-backend",
   "attributes": {
-    "backgroundColor": { "type": "string" },
```
Comment (mleray) [inline assets/src/block-templates/page-header/block.json:9]: This was not used

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [phobos](https://www-dev.greenpeace.org/test-phobos) | [admin](https://www-dev.greenpeace.org/test-phobos/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-phobos/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-phobos) | [composer-local.json](https://github.com/greenpeace/planet4-test-phobos/blob/main/composer-local.json)

:watch: 2023.06.20 09:19:35

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [titan](https://www-dev.greenpeace.org/test-titan) | [admin](https://www-dev.greenpeace.org/test-titan/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-titan/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-titan) | [composer-local.json](https://github.com/greenpeace/planet4-test-titan/blob/main/composer-local.json)

:watch: 2023.06.21 08:18:06

File: assets/src/block-templates/campaign/template.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    ['planet4-block-templates/side-image-with-text-and-cta', {
+      title: __('The solution', 'planet4-blocks'),
+      mediaPosition: 'right',
+    }],
+    ['core/spacer', {height: '32px'}],
+    ['planet4-blocks/covers', {
+      title: __('How you can help', 'planet4-blocks'),
+    }],
+    ['core/spacer', {height: '48px'}],
+    ['gravityforms/form'],
```
[the remark this answers is not in the document]
Comment (Osong-Michael) [reply assets/src/block-templates/campaign/template.js:22]: It is merged already, she can rebase an use the template.

[the remark this answers is not in the document]
Comment (mleray) [reply assets/src/block-templates/campaign/template.js:24]: Updated! I've included a small refactor of the `gravityFormWithText` function, let me know what you think 🙂

[the remark this answers is not in the document]
Comment (lithrel) [reply assets/src/block-templates/campaign/template.js:24]: no need to have 2 forms then :grin:  (l.24)

**Review by lithrel — APPROVED** [review]
one form to remove and good for me :+1:

File: assets/src/block-templates/campaign/template.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (mleray) [reply assets/src/block-templates/campaign/template.js]: Ugh 🙈 updated

**Pull request merged**